log/async_file_writer_test.go

```diff
@@ -1,24 +1,41 @@
 package log
 
 import (
-	"io/ioutil"
 	"os"
 	"strings"
 	"testing"
 )
 
-func TestWriter(t *testing.T) {
-	w := NewAsyncFileWriter("./hello.log", 100)
+func TestWriterHourly(t *testing.T) {
+	w := NewAsyncFileWriter("./hello.log", 100, false)
 	w.Start()
 	w.Write([]byte("hello\n"))
 	w.Write([]byte("world\n"))
 	w.Stop()
-	files, _ := ioutil.ReadDir("./")
+	files, _ := os.ReadDir("./")
```

Comment:
Why os instead of already existing ioutil?

Reply:
`ioutil.ReadDir` is deprecated since go 1.16

From `ioutil` package:

```
// Deprecated: As of Go 1.16, os.ReadDir is a more efficient and correct choice:
// it returns a list of fs.DirEntry instead of fs.FileInfo,
// and it returns partial results in the case of an error
// midway through reading a directory.
```